### PR TITLE
Enable bundle size analysis

### PR DESCRIPTION
**Pull request by denniskigen**
Adds ability to analyze app bundle size using [webpack bundle analyzer](https://www.npmjs.com/package/webpack-bundle-analyzer) and provides a simple script for this purpose.

Screenshot 2020-09-16 at 14 47 20

Usage:
- Run `npm run build-prod:stats` to generate a production build with bundle size statistics.
- Launch the bundle analyzer by running `npm run stats`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,10 +12,12 @@
     "lint": "ng lint",
     "e2e": "ng e2e",
     "build-prod": "node version.js && bash ./scripts/ng.sh build --prod --aot --optimization=false",
+    "build-prod:stats": "npm run build-prod -- --statsJson=true",
     "serve:prod": "http-server dist/ngx-amrs/ -P https://ngx.ampath.or.ke",
     "prebuild": "node version.js",
     "prestart": "node version.js",
-    "prettier:check": "prettier --check src/app"
+    "prettier:check": "prettier --check src/app",
+    "stats": "webpack-bundle-analyzer dist/ngx-amrs/stats.json"
```
Comment (kipchubett) [inline package.json]: What if you combine the two commands. Just thinking aloud 🤔
`"stats": "npm run build-prod -- --statsJson=true && webpack-bundle-analyzer dist/ngx-amrs/stats.json"`

**Review by kipchubett — DISMISSED** [review]
LGTM!
Thanks @denniskigen

**Review by donaldkibet — DISMISSED** [review]
LGTM :rocket:

File: package.json (hunk 1 above)
Comment (denniskigen) [reply package.json]: Sounds good, updated @corneliouzbett

**Review by kipchubett — APPROVED** [review]

**Pull request merged**